stardoc/stardoc.bzl

```diff
@@ -152,5 +156,25 @@ non-default semantic flags required to use the given Starlark symbols.
             cfg = "host",
             executable = True,
         ),
+        "header_template": attr.label(
+            doc = "The input file template for header generated in documentation.",
+            allow_single_file = [".vm", ".txt"],
```

Comment:
Whoops, we didn't resolve this discussion offline -- if you can truly only find that .vm templates are used by velocity engine, maybe lets restrict this just to ".vm". (and for the other template attributes)